### ISSUE TITLE
[BUG] link sticker not visible

### ISSUE THREAD

**Issue by ptakptakptak** [adw0rd/instagrapi#549]
Should a visible link sticker appear on the media after adding an instagram story with a link?
There is a clickable area on the media but no visible sticker appear after publishing.

It does appear for countdown sticker.
Should it appear for link sticker as well or should I add it in manually in external editor?

**Comment by adw0rd** [comment]
> Should a visible link sticker appear on the media after adding an instagram story with a link?
No, the video editor embed into the Instagram app will do this for you before uploading to the Instagram server

> There is a clickable area on the media but no visible sticker appear after publishing.
Yes, the Instagram app and Instagrapi will make the markup for the sticker area

> Should it appear for link sticker as well or should I add it in manually in external editor?
You can try using [StoryBuilder](https://github.com/adw0rd/instagrapi/blob/master/instagrapi/story.py#L120-L155)

https://adw0rd.github.io/instagrapi/usage-guide/story.html
